app/views/simulator/edit.html.erb

```diff
@@ -16,7 +16,7 @@
   <button class="navbar-toggler collapsed" type="button" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-controls="collapsedNavbar" aria-expanded="false" aria-label="Toggle navigation">
     <span class="navbar-toggler-icon"></span>
   </button>
-  <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+  <div class="collapse navbar-collapse" style="justify-content: space-between;" id="bs-example-navbar-collapse-1">
```

Comment:
No inline style please

Reply:
Alright. I thought it was allowed to use. 
It's resolved now using the bootstrap class. Can you please review it again?